src/context/loadCustomVolumes.tsx

```diff
@@ -18,8 +19,11 @@ export const loadCustomVolumes = async (
       const contentTypes = ["filesystem", "block"];
       const isFilesystemOrBlock = contentTypes.includes(volume.content_type);
       const isCustom = volume.type === "custom";
+      // Ceph pool volumes cannot be shared by multiple instances
+      const isCephDriverAndUsed =
+        pool.driver === cephDriver && volume.used_by?.length;
```

Comment:
I put the logic here so that we don't have to iterate the returned result for filtering. do you think it's better to just do a filter in `CustomVolumeSelectModal`?

Reply:
on second thought, I think it's better to keep logic where it belongs, will move it to the component

Reply:
We could keep it here, but then the function name should announce what it does. From the current name, the filter is an unexpected side effect.